.rhcicd/pr_check.sh

```diff
@@ -20,6 +20,7 @@ buildAndPushToQuay "notifications-connector-drawer"
 buildAndPushToQuay "notifications-connector-email"
 buildAndPushToQuay "notifications-connector-google-chat"
 buildAndPushToQuay "notifications-connector-microsoft-teams"
+buildAndPushToQuay "notifications-connector-pagerduty"
```

Comment:
It's too early for an update of `pr_check.sh` because the deployment config has not been added yet in app-interface. Your changes are the good ones, but they'll need to be done later.
```suggestion
```